DOC 5.0.0 changelog removed and changed API (#150)
silverstripe/developer-docs: en/05_Contributing/04_Release_Process.md

* Add the following docblock `@deprecated 1.2.3 Use AnotherClass instead`
* Add `Deprecation::notice('1.2.3', 'Use AnotherClass instead', SCOPE_CLASS);` to the top of `__construct()`

> * Most of the time you'll want to wrap `Deprecation::notice()` with `Deprecation::withNoReplacement()` as there's usually no replacement for deprecated classes. e.g.

Reviewer: ```suggestion
* Wrap `Deprecation::notice()` with `Deprecation::withNoReplacement()` if there's no replacement for that deprecated class. e.g:
```
Some classes will have replacements. This suggestion makes it very clear what to wrap.

Author: Updated
